example/Proofs/ConsUncons.hs

```diff
@@ -0,0 +1,23 @@
+-- | This example demonstrates how to use the translation of an error
+--   throwing function to prove properties about this function and about the
+--   possible errors that can be thrown.
+--
+--   This example uses an error throwing implementation of the @uncons@
+--   function.
+
+module Proofs.ConsUncons where
+
+import           Test.QuickCheck
+
+-- First we have to define the @uncons@ function in an error throwing way.
```

Comment:
In the comments and in the error message you wrote `uncons` instead of `unconsE`.